lib/handlers/quote/handler.ts

```diff
@@ -383,14 +384,30 @@ export function compareQuotes(lhs: Quote, rhs: Quote, tradeType: TradeType): boo
 const getQuotedAmount = (quote: Quote, tradeType: TradeType) => {
   if (tradeType === TradeType.EXACT_INPUT) {
     if (quote.routingType === RoutingType.CLASSIC) {
+      // TODO: check with the team whether they'd like to compare amountOutGasAndPortionAdjusted instead
```

Comment:
i think almost certainly here we want to go with what you currently have on

Reply:
think we aligned on using `amountOutGasAndPortionAdjusted`